HelpMyStreetFE/HelpMyStreetFE/Views/Shared/DisplayTemplates/RequestSummaryViewModel.cshtml

```diff
@@ -16,8 +16,25 @@
     bool singleJob = Model.Item.JobBasics.Count() == 1 && Model.Item.RequestType.Equals(RequestType.Task);
     var jobId = singleJob ? Base64Utils.Base64Encode(Model.Item.JobBasics.First().JobID) : "";
 
+    var hmsRef = Model.Item.JobSummaries.First().HmsReference.Split("-");
+    var requestHMSReference = String.Join("-", hmsRef.Take(hmsRef.Count() - 1));
+
     var className = "";
     JobSummary nextTaskDue = null;
+    var location = "";
+
+    if (admin)
+    {
+        location = Model.Item.PostCode;
+    }
+    else if (Model.Item.JobSummaries.Exists(j => j.JobStatus == JobStatuses.Open))
```

Comment:
This doesn't look quite right.  I would think an Open job is when we _would_ want the distance.

That said, I think in this branch RequestSummary will only be used in the Admin view.  Maybe just drop the `else if` block here and keep...

if (admin) { NG1 6DQ }
else { NG1 (6 miles away) }